aea/helpers/dialogue/base.py

```diff
@@ -180,11 +189,9 @@ def __init__(
         ) -> None:
             """
             Initialize a dialogue.
-
```

Comment:
We use the pattern to separate first line and params throughout. It is not good practice to change formatting of code you are not touching. Just leave it as it is and focus on your own contributions.

Reply:
There is no change. Not sure why GH shows a removed line.